pyTigerGraph/gds/gsql/featurizer/tg_label_prop.gsql

```diff
@@ -0,0 +1,66 @@
+CREATE QUERY tg_label_prop (SET<STRING> v_type, SET<STRING> e_type, INT max_iter=10, INT output_limit=10, 
+  BOOL print_accum = TRUE, STRING file_path = "", STRING attr = "")  SYNTAX V1 {
```

Comment:
Could you try if V2 works? If so let's use V2 as the database is pushing for V2

Reply:
Yes, it is working I just changed it